### PR TITLE
Feature - #3031 - search for org repos

### PR DESCRIPTION
**Pull request by richmahn**
Adds the search form to the Org home page like it is for the User home page to search and sort repos of the Org. Addresses #3031 

Credit goes to @techknowlogick who made most all these edits in the closed PR #3081. Only updates I made were for the new sort options and new way to handle errors.

To test:
Check out branch, generate build, run gitea web, make a new org in your instance of Gitea and add multiple repos, staring and forking some of them. Then use the Search and Sort form on the Org home page.

### CODE REVIEW

**Comment by techknowlogick** [comment]
Thanks for working on this 😄 

I'd give this code 5/5 ⭐️s, however since I was involved in this I'll abstain from reviewing.

File: routers/user/home.go [hunk 1]
```diff
@@ -416,13 +451,33 @@ func showOrgProfile(ctx *context.Context) {
...
-		if err != nil {
-			ctx.ServerError("GetRepositories", err)
-			return
+		if len(keyword) == 0 {
+			repos, err = models.GetUserRepositories(org.ID, showPrivate, page, setting.UI.User.RepoPagingNum, orderBy.String())
+			if err != nil {
+				ctx.ServerError("GetRepositories", err)
+				return
+			}
+			ctx.Data["Repos"] = repos
```
Comment (lafriks) [inline routers/user/home.go]: Assignment to CTX.data from both if/else could be moved lover after if/else block

**Comment by lunny** [comment]
@richmahn thanks for your PR. Could you give some screenshots?

File: routers/user/home.go (hunk 1 above)
Comment (richmahn) [reply routers/user/home.go]: Good catch. I do prefer all those things to be set together at the end. Done.

**Comment by richmahn** [comment]
@lunny Screenshots:

**Org home page without any search and default sort of recently updated**
screen shot 2019-02-07 at 6 06 28 am

**Org home search for `fr_` with default sorting of recently updated**
screen shot 2019-02-07 at 6 05 32 am

**Org home search for `fr_` with sort of most forks**
screen shot 2019-02-07 at 6 05 49 am

**Org home search for `fr_` with sort of most stars**
screen shot 2019-02-07 at 6 06 06 am

**Comment by richmahn** [comment]
@lunny and other reviewers, I will leave it running at http://bg.door43.org/unfoldingword for now if that helps

**Comment by richmahn** [comment]
Ok, ready to merge (fixed the retaining of the sort order when changing the search query)

**Review by lafriks — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**